### ISSUE TITLE
Got Error: AttributeError: 'ArqRedis' object has no attribute 'wait_closed'

### ISSUE THREAD

**Issue by jockerz** [ninoseki/arq-dashboard#10]
Hi @ninoseki 
I just dicovered and tried this cool project but got some errors.

Please take a look and tell me anything about this bug.

**`pip freeze`**
```
aiometer==0.3.0
aioredis==1.3.1
anyio==3.6.1
arq==0.22
# Also got same error on `0.1.3`
-e git+https://github.com/ninoseki/arq-dashboard@9ce71b7aeab59db2cbc027338487ba4dc7eb7f36#egg=arq_dashboard
arrow==1.2.3
asgiref==3.5.2
async-cache==1.1.1
async-timeout==4.0.2
click==8.1.3
Deprecated==1.2.13
fastapi==0.75.2
h11==0.14.0
hiredis==2.0.0
httptools==0.5.0
idna==3.4
loguru==0.6.0
packaging==21.3
pydantic==1.10.2
pyhumps==3.7.3
pyparsing==3.0.9
python-dateutil==2.8.2
python-dotenv==0.21.0
PyYAML==6.0
redis==4.3.4
six==1.16.0
sniffio==1.3.0
starlette==0.17.1
typing_extensions==4.4.0
uvicorn==0.17.6
uvloop==0.17.0
watchgod==0.8.2
websockets==10.3
wrapt==1.14.1
```

Here is the log:

```
INFO:     127.0.0.1:42158 - "GET /js/app.ec3a003e.js HTTP/1.1" 304 Not Modified
/dev/arq/arq-dashboard/arq_dashboard/dependencies.py:19: RuntimeWarning: coroutine 'Redis.close' was never awaited
  redis.close()
RuntimeWarning: Enable tracemalloc to get the object allocation traceback
INFO:     127.0.0.1:42132 - "GET /api/stats/ HTTP/1.1" 500 Internal Server Error
ERROR:    Exception in ASGI application
Traceback (most recent call last):
  File "/dev/arq/arq-dashboard/arq_dashboard/dependencies.py", line 16, in get_redis
    yield redis
  File "/dev/arq/arq-dashboard/arq_dashboard/queue.py", line 106, in get_stats
    job_ids = await get_job_ids(redis, self.name)
  File "/dev/arq/arq-dashboard/arq_dashboard/queue.py", line 40, in get_job_ids
    job_ids = set(await redis.zrangebyscore(queue_name))
TypeError: zrangebyscore() missing 2 required positional arguments: 'min' and 'max'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/dev/arq/venv/lib/python3.9/site-packages/uvicorn/protocols/http/httptools_impl.py", line 372, in run_asgi
    result = await app(self.scope, self.receive, self.send)
  File "/dev/arq/venv/lib/python3.9/site-packages/uvicorn/middleware/proxy_headers.py", line 75, in __call__
    return await self.app(scope, receive, send)
  File "/dev/arq/venv/lib/python3.9/site-packages/fastapi/applications.py", line 261, in __call__
    await super().__call__(scope, receive, send)
  File "/dev/arq/venv/lib/python3.9/site-packages/starlette/applications.py", line 112, in __call__
    await self.middleware_stack(scope, receive, send)
  File "/dev/arq/venv/lib/python3.9/site-packages/starlette/middleware/errors.py", line 181, in __call__
    raise exc
  File "/dev/arq/venv/lib/python3.9/site-packages/starlette/middleware/errors.py", line 159, in __call__
    await self.app(scope, receive, _send)
  File "/dev/arq/venv/lib/python3.9/site-packages/starlette/middleware/gzip.py", line 23, in __call__
    await responder(scope, receive, send)
  File "/dev/arq/venv/lib/python3.9/site-packages/starlette/middleware/gzip.py", line 42, in __call__
    await self.app(scope, receive, self.send_with_gzip)
  File "/dev/arq/venv/lib/python3.9/site-packages/starlette/exceptions.py", line 82, in __call__
    raise exc
  File "/dev/arq/venv/lib/python3.9/site-packages/starlette/exceptions.py", line 71, in __call__
    await self.app(scope, receive, sender)
  File "/dev/arq/venv/lib/python3.9/site-packages/fastapi/middleware/asyncexitstack.py", line 21, in __call__
    raise e
  File "/dev/arq/venv/lib/python3.9/site-packages/fastapi/middleware/asyncexitstack.py", line 18, in __call__
    await self.app(scope, receive, send)
  File "/dev/arq/venv/lib/python3.9/site-packages/starlette/routing.py", line 656, in __call__
    await route.handle(scope, receive, send)
  File "/dev/arq/venv/lib/python3.9/site-packages/starlette/routing.py", line 259, in handle
    await self.app(scope, receive, send)
  File "/dev/arq/venv/lib/python3.9/site-packages/starlette/routing.py", line 61, in app
    response = await func(request)
  File "/dev/arq/venv/lib/python3.9/site-packages/fastapi/routing.py", line 227, in app
    raw_response = await run_endpoint_function(
  File "/dev/arq/venv/lib/python3.9/site-packages/fastapi/routing.py", line 160, in run_endpoint_function
    return await dependant.call(**values)
  File "/dev/arq/arq-dashboard/arq_dashboard/api/endpoints/stats.py", line 25, in get_stats
    return await _get_stats(queue_name)
  File "/dev/arq/venv/lib/python3.9/site-packages/cache/async_ttl.py", line 56, in wrapper
    self.ttl[key] = await func(*args, **kwargs)
  File "/dev/arq/arq-dashboard/arq_dashboard/api/endpoints/stats.py", line 15, in _get_stats
    return await queue.get_stats()
  File "/dev/arq/arq-dashboard/arq_dashboard/queue.py", line 110, in get_stats
    statuses = await aiometer.run_all(
  File "/usr/lib/python3.9/contextlib.py", line 193, in __aexit__
    await self.gen.athrow(typ, value, traceback)
  File "/dev/arq/arq-dashboard/arq_dashboard/dependencies.py", line 20, in get_redis
    await redis.wait_closed()
AttributeError: 'ArqRedis' object has no attribute 'wait_closed'
```

**Comment by ninoseki** [comment]
Hello, thank you for reporting the issue.
I believe the issue is fixed by #11. And I just released a new version including the change.
https://pypi.org/project/arq-dashboard/0.1.4/

**Comment by jockerz** [comment]
Great to hear so.

Tested the new version and it works so well.

Thank you :)
